cypress/integration/use-cases/language-selection.spec.js

```diff
@@ -5,9 +5,9 @@ describe('switching languages', () => {
         cy.get('footer .i18n-widget-language select').select('German (Deutsch)', {
             force: true,
         });
-        cy.contains('Changing the language will redirect you to another domain.');
+        cy.contains('Wenn Du die Sprache wechelst, wirst Du zu einer anderen Domain weitergeleitet.');
```

Comment:
Need to add route for german -> english, also both text should appear, what if the user clicks the wrong language by accident?